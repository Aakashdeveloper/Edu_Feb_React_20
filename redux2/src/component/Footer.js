import React from 'react';

const Footer = () => {
    return(
        <footer>
            <center>
                &copy; Edureka
            </center>
        </footer>
    )

}

export default Footer