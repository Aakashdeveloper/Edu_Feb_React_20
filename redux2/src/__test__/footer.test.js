import React from 'react';
import Footer from '../component/Footer';
import {create} from 'react-test-renderer';

describe('Snapshot test',() => {
    test('Testing my footer component',()=>{
        let tree = create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})