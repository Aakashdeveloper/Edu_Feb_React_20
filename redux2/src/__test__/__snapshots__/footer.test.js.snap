// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Testing my footer component 1`] = `
<footer>
  <center>
    Â© Edureka
  </center>
</footer>
`;
